components/Input/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, InputHTMLAttributes, MouseEventHandler, ChangeEventHandler } from 'react';
+import { InputHTMLAttributes, MouseEventHandler, ChangeEventHandler } from 'react';
```

Comment:
```suggestion
import { Component, InputHTMLAttributes, MouseEventHandler, ChangeEventHandler } from 'react';
```

Reply:
cannot adjust children if parent component is a class component